<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Distance Traveled by Gain</string>
  </property>
  <widget class="QPushButton" name="pushButtonCalc">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetOut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>531</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
